### PR TITLE
feat: Add _getOpenedPRs for BitBucket

### PR DESCRIPTION
**Pull request by jerryzhang222**
Add getOpenedPRs

Related: https://github.com/screwdriver-cd/screwdriver/issues/417

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -646,13 +646,43 @@ class BitbucketScm extends Scm {
     }
 
     /**
+    * Get list of opened PRs of a pipeline
```
Comment (d2lam) [inline index.js]: Get list of objects which consists of opened PR names and its ref

File: test/index.test.js [hunk 2]
```diff
@@ -1458,4 +1458,43 @@ describe('index', function () {
...
+        const oauthToken = 'oauthToken';
+        const scmUri = 'hostName:repoId:branchName';
+
+        beforeEach(() => {
+            requestMock.yieldsAsync(null, {
+                statusCode: 200
+            });
+        });
+
+        it('works', () => {
```
Comment (d2lam) [inline test/index.test.js]: Some better description for the test?

File: test/index.test.js [hunk 3]
```diff
@@ -1458,4 +1458,43 @@ describe('index', function () {
             });
         });
     });
+
+    describe('_getOpenedPRs', () => {
+        const oauthToken = 'oauthToken';
+        const scmUri = 'hostName:repoId:branchName';
+
+        beforeEach(() => {
```
Comment (d2lam) [inline test/index.test.js]: Don't need this if it's just 1 test case anyway

File: index.js [hunk 4]
```diff
@@ -646,13 +646,43 @@ class BitbucketScm extends Scm {
...
     /**
+    * Get list of objects (each consists of opened PR name and ref (branch)) of a pipeline
+    * @method getOpenedPRs
+    * @param  {Object}   config              Configuration
+    * @param  {String}   config.scmUri       The scmUri to get opened PRs
+    * @param  {String}   config.token        The token used to authenticate to the SCM
+    * @return {Promise}
+    */
+    _getOpenedPRs(config) {
+        const repoId = getScmUriParts(config.scmUri)[1];
```
Comment (FenrirUnbound) [inline index.js]: `1` is a magic number

**Pull request merged**